vulns/ajenti-panel/PYSEC-0000-CVE-2018-1000080.yaml

```diff
@@ -0,0 +1,25 @@
+id: PYSEC-0000-CVE-2018-1000080
+details: Ajenti version version 2 contains a Insecure Permissions vulnerability in
+  Plugins download that can result in The download of any plugins as being a normal
+  user. This attack appear to be exploitable via By knowing how the requisition is
+  made, and sending it as a normal user, the server, in response, downloads the plugin.
+affected:
+- package:
+    name: ajenti-panel
+    ecosystem: PyPI
+    purl: pkg:pypi:ajenti-panel
+  ranges:
+  - type: ECOSYSTEM
+    events:
+    - introduced: "0"
+references:
+- type: WEB
+  url: https://medium.com/stolabs/security-issues-on-ajenti-d2b7526eaeee
+- type: PACKAGE
+  url: https://pypi.org/project/ajenti-panel
+- type: ADVISORY
+  url: https://nvd.nist.gov/vuln/detail/CVE-2018-1000080
+aliases:
+- CVE-2018-1000080
```

Comment:
This is pretty old... Is this still unfixed as of today?

Reply:
I haven't found anything that references any of them being fixed (or any acknowledgement of them whatsoever).

Reply:
I suppose when I get some more time I could try reproducing the steps from the blog post on the latest version to see if it still works and then work backwards or something.  There are quite a few other packages that still don't show a fixed version that I need to investigate as well.

Reply:
Sounds great to me, thanks!